spec.md

```diff
@@ -347,36 +341,36 @@ a trailing `'`{.firrtl}.  The following is an example of a raw string literal:
 'world'
 ```
 
-## String-encoded Integer Literal
+## Radix-specified Integer Literal
 
-A string-encoded integer literal is a special string literal with one of the
+A radix-specified integer literal is a special integer literal with one of the
 following leading characters to indicate the numerical encoding:
 
 - `b`{.firrtl} -- for representing binary numbers
 - `o`{.firrtl} -- for representing octal numbers
 - `h`{.firrtl} -- for representing hexadecimal numbers
 
-Signed string-encoded integer literals have their sign following the leading
+Signed radix-specified integer literals have their sign following the leading
 encoding character.
 
 The following string-encoded integer literals all have the value `42`:
 
 ``` firrtl
-"b101010"
-"o52"
-"h2a"
+b101010
+o52
+h2a
```

Comment:
I have a slight preference for the leading-0 notation since it seems like a future proofing against identifiers/parameters? WDYT?

Reply:
I like it. Going with a leading `0` helps future proof things for when parameters inevitably show up.

I won't update this PR just yet as it may be better to have these discussions on https://github.com/chipsalliance/firrtl-spec/issues/89 as opposed to me thrashing this PR back and forth.

E.g., I would like to push either in the direction of no radix-encoded integer literals (decimal everywhere) or only use hex-encoded integer literals for constant integer expressions and decimal everywhere else.

Reply:
Switched to `0b`/`0o`/`0h`.